### DISCUSSION TITLE
difference between "Vanilla"and "Test app"

### DISCUSSION THREAD

**Discussion by Alexufo** [callstack/react-native-builder-bob#651] [Q&A] [answered]
Kotlin & Objective-C

What type of example app do you want to create? › Test app or Vanilla

what's the difference?

**Comment by kraenhansen** [comment, marked as answer]
- "Vanilla" use the `@react-native-community/cli` https://github.com/callstack/react-native-builder-bob/blob/9c5638a224d558071ae01f010da8b7a6860f00ee/packages/create-react-native-library/src/exampleApp/generateExampleApp.ts#L83-L97
- and "Test app" use [the `react-native-test-app` package](https://github.com/microsoft/react-native-test-app) https://github.com/callstack/react-native-builder-bob/blob/9c5638a224d558071ae01f010da8b7a6860f00ee/packages/create-react-native-library/src/exampleApp/generateExampleApp.ts#L65-L81
